kibana/config/entrypoint.sh

```diff
@@ -23,4 +23,7 @@ sleep 5
 
 ./kibana_settings.sh &
 
+
+./xpack_config.sh &
+
```

Comment:
This script must be executed before KIbana starts. In theory it does, but it is not necessary to launch it in the background.